eagle-stores/src/App.js

```diff
@@ -1,9 +1,31 @@
 import "./App.css";
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import NavBar from "./components/NavBar/NavBar"
+import Carousel from "./components/Carousel/Carousel"
+import HomeCategory from "./components/HomeCategory/HomeCategory";
+import Footer from "./components/Footer/Footer";
+import Mockman from "mockman-js";
+
+function MockAPI() {
+  return (
+    <div className="MockAPI">
+      <Mockman />
+    </div>
+  );
+}
 
 function App() {
   return (
    <>
-   <h1>EagleStores</h1>
+   <BrowserRouter>
+    <NavBar />
+    <Carousel />
```

Comment:
Carousel and HomeCategory you can put inside Home component

Reply:
Done. Create home file inside pages and put carousel and categories inside that